### ISSUE TITLE
Boundary Conditions

### ISSUE THREAD

**Issue by StanleyN1** [JuliaMolSim/Molly.jl#167]
I was wondering if Molly had alternative boundary conditions to periodic such as reflecting boundary conditions?

How are pairwise interactions calculated if there are no Ewald sums implemented? Are the pairwise distances in the original simulated box used in calculating the Coulombic potential?

**Comment by jgreener64** [comment]
No, at the minute just periodic (cube/triclinic) and infinite boundaries.

We use the minimum image convention to calculate the closest distance from the 27 adjacent periodic images, see `vector`.
